Jenkinsfile

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env groovy
+
+/**
+ * Build Contiki Applications
+ *
+ */
+
+
+node('docker && imgtec') {
+    def docker_image
+    docker_image = docker.image "imgtec/creator-builder:latest" // TODO for now using the creator one
+    docker_image.inside {
+        stage('Prepare') {
+            checkout([$class: 'GitSCM',
+                userRemoteConfigs: scm.userRemoteConfigs,
+                branches: scm.branches,
+                doGenerateSubmoduleConfigurations: scm.doGenerateSubmoduleConfigurations,
+                submoduleCfg: scm.submoduleCfg,
+                browser: scm.browser,
+                gitTool: scm.gitTool,
+                extensions: scm.extensions + [
+                    [$class: 'CleanCheckout'],
+                    [$class: 'PruneStaleBranch'],
+                ],
+            ])
+        }
+        stage('Build') {
+            sh 'git submodule init "dev/ca8210"'
+            sh 'git submodule update'
+            sh 'cd examples/hello-world && make TARGET=mikro-e USE_CA8210=1 CHANNEL=26 PAN_ID=0xabcd USE_SERIAL_PADS=1'
+            sh 'cd examples/hello-world && xc32-bin2hex hello-world.mikro-e'
+            sh 'cd examples/ && git clone https://github.com/mtusnio/LetMeCreateIoT.git'
+            sh 'cd examples/LetMeCreateIoT/ && ./install.sh -p ../../'
+            sh 'cd examples/LetMeCreateIoT/examples/accel/ && make && xc32-bin2hex main.mikro-e && cp main.hex ../../../../'
+            sh 'ls -al'
+       }
+        stage('Upload') {
+            sh 'cp examples/hello-world/hello-world.hex .'
```

Comment:
no need of this line, you can simply do this

    archiveArtifacts '**/hello-world/hello-world.hex'